pkgs/development/tools/scalafix/default.nix

```
@@ -1,47 +1,49 @@
-{ lib, stdenv, jre, coursier, makeWrapper, installShellFiles, setJavaClassPath }:
-
-let
-  baseName = "scalafix";
-  version = "0.10.0";
+{
+  lib,
+  stdenv,
+  jre,
+  coursier,
+  makeWrapper,
+  installShellFiles,
+  setJavaClassPath,
+}:
+stdenv.mkDerivation rec {
```

An alternative here, to make overriding slightly easier, is to use [the function variant](https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes):

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

If you do this you also need to change `pname` to `finalAttrs.pname` below.